gf-button/standard-button.md

```diff
@@ -75,11 +73,11 @@ GFButton(
  ),
 ```
 
-### Full Width Button
+### Bot&oacute;n de ancho m&aacute;ximo
 
-![Flutter Full Width Button](../.gitbook/assets/full-width-button-2x.png)
+![Flutter Bot&oacute;n de ancho m&aacute;ximo](../.gitbook/assets/full-width-button-2x.png)
 
-Full Width button specify how wide the button should be. By setting **fullWidthButton** state `true`, it will change the button to a Full-width button with square corners and no border on the left or right. Default **fullWidthButton** set to `false`.
```

Comment:
It says square corners, I think it should be rounded corners, like block button, am I right?